web/src/views/Home.scss

```diff
@@ -28,3 +28,11 @@
 .recent-searches {
   display: flex;
 }
+
+.no-results {
+  text-align: center;
+}
+
+.no-results h1 {
+  font-size:580%;
```

Comment:
nit: space between the colon and the value.

Reply:
> Thanks for this! Just a minor edit: please keep `br`/`hr` elements the way they were (in "Home.vue"). `<br>` is sufficient in html and need those to be consistent across all files.

Oh sorry thought i ignored that change, I ran lint by mistake so it did some changes.